ak_vendor/enums.py

```diff
@@ -455,3 +455,15 @@ class Meta:
         NONE = [0, "None"]
         TOTP = [1, "TOTP"]
         HOTP = [2, "HOTP"]
+
+
+@choices
+class FileTypeEnum:
```

Comment:
Can we use `FileFormatEnum`? (reasoning: it represents package file format. Type can be later used for some internal categorization in future)

Reply:
Good idea.